measure/Cargo.toml

```diff
@@ -19,5 +19,12 @@ solana-metrics = { path = "../metrics", version = "1.6.0" }
 jemallocator = "0.3.2"
 jemalloc-ctl = "0.3.2"
 
+[features]
+# mainly for heaptrack, which seems to be incompatible with jemalloc
+# opt-out ("no-" prefix) is chosen for convenience for the default case
+# (= with jemalloc), considering complexies of proper opt-in (default)
+# feature plumbing
+no-jemalloc = []
```

Comment:
side note: arguably, the most mature pluggable malloc system on the rust land will be this one: https://github.com/baiyuqing/jiehuafa/blob/034fe619f0f8d9a4dc14edcf2b36bcca64e5be6c/tikv/components/tikv_alloc/src/lib.rs#L43

We're far from it but hope to be there in future to squeeze out every bits out of given computing system... :)